packages/client/src/router/index.js

```diff
@@ -29,7 +29,7 @@ export const routes = [
         // Redirect any old hash-style URLs to the new history API URL.
         return { path: to.hash.substring(1), hash: '' };
       }
-      return 'my-grants';
```

Comment:
While we're here, I'd love to switch this to a named route (`return { name: 'grants' }`) instead of a relative path, since using paths (especially relative ones) is a bit more brittle. https://router.vuejs.org/guide/essentials/redirect-and-alias.html